docs/bluetooth.md

```diff
@@ -126,7 +126,7 @@ cancel = bluetooth.async_track_unavailable(hass, _unavailable_callback, "44:44:3
 
 ### Fetching the bleak `BLEDevice` from the `address`
 
-Integrations wishing to avoid the overhead of starting an additional scanner to resolve the address may call the `bluetooth.async_ble_device_from_address` API, which returns a `BLEDevice` if the `bluetooth` integration scanner has recently seen the device. The integration MUST fall back to connecting via the `address` if the `BLEDevice` is unavailable.
+Integrations should avoid the overhead of starting an additional scanner to resolve the address may call the `bluetooth.async_ble_device_from_address` API, which returns a `BLEDevice` if the `bluetooth` integration scanner has recently seen the device. The integration MUST fall back to connecting via the `address` if the `BLEDevice` is unavailable.
```

Comment:
This doesn't sound right:
> Integrations should avoid the overhead of starting an additional scanner to resolve the address may call the

Reply:
Also the note that they MUST fall back is wrong now as well.  If it doesn't return the device they won't be able to reach it no matter how hard they try as the underlying stack can't see it

Reply:
Adjusted